AnnService/inc/Core/Common/BKTree.h

```diff
@@ -255,7 +262,9 @@ namespace SPTAG
                 }
                 args.ClearCounts();
                 args.ClearDists(MaxDist);
-                currDist = KmeansAssign(data, indices, first, batchEnd, args, false, 0);
```

Comment:
why this line need to change?

Reply:
I don't think it does. It looks like I added it last summer, but I do not remember why. I will run a recall test without the change and check for sure.

- [ ]  Recall test w/o KmeansAssign baseSquare
- [ ] Check for other unnecessary baseSquare changes

Reply:
Recall matches without the change. I think it was added by mistake with the other normalization changes. Should be fixed in latest version